src/components/v3/BottomTicker.tsx

```
@@ -0,0 +1,115 @@
+      style={{
+        background: "var(--bg-panel)",
+        borderColor: "var(--border-subtle)",
+      }}
+    >
+      {/* Left: Agent health dots */}
+      <div className="flex items-center gap-2 flex-shrink-0">
+        {healthDots.map((dot) => (
+          <motion.div
+            key={dot.id}
+            animate={{ opacity: [0.5, 1, 0.5] }}
+            transition={{ duration: 2, repeat: Infinity, delay: Math.random() }}
```

ESLint

[error] Error: Cannot call impure function during render

`Math.random` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/components/v3/BottomTicker.tsx:78:65
  76 |             key={dot.id}
  77 |             animate={{ opacity: [0.5, 1, 0.5] }}
> 78 |             transition={{ duration: 2, repeat: Infinity, delay: Math.random() }}
     |                                                                 ^^^^^^^^^^^^^ Cannot call impure function
  79 |             className="w-1.5 h-1.5 rounded-full"
  80 |             style={{ background: dot.color }}
  81 |           />

(react-hooks/purity)